coalib/coala_modes.py

```diff
@@ -26,7 +29,8 @@ def mode_normal(console_printer, log_printer, args, debug=False):
         console_printer=console_printer,
         args=args,
         debug=debug)
-
+    if log_printer:
+        logging.warning('Warning: The use of log_printer is deprecated')
```

Comment:
And also you're using if statement in the end of the function it would be good if you use it after `import logging` in the beginning of function and show the warning there :+1: